### ISSUE TITLE
"Blowfish deprecated"

### ISSUE THREAD

**Issue by neilyoung** [ansible/ansible#77762]
### Summary

ansible --version spits a warning about a "blowfish deprecation". Most likely caused by an upstream package (paramiko), see their issue https://github.com/paramiko/paramiko/issues/2038

### Issue Type

Bug Report

### Component Name

??

### Ansible Version

```console
$ ansible --version
ansible --version
/opt/homebrew/Cellar/ansible/5.7.0/libexec/lib/python3.10/site-packages/paramiko/transport.py:236: CryptographyDeprecationWarning: Blowfish has been deprecated
  "class": algorithms.Blowfish,
ansible [core 2.12.5]
  config file = None
  configured module search path = ['/Users/decades/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/homebrew/Cellar/ansible/5.7.0/libexec/lib/python3.10/site-packages/ansible
  ansible collection location = /Users/decades/.ansible/collections:/usr/share/ansible/collections
  executable location = /opt/homebrew/bin/ansible
  python version = 3.10.4 (main, Apr 26 2022, 19:36:29) [Clang 13.1.6 (clang-1316.0.21.2)]
  jinja version = 3.1.2
  libyaml = True
~ $
```

### Configuration

```console
$ ansible-config dump --only-changed
```

### OS / Environment

macOS 12.3, Apple silicon

### Steps to Reproduce

```yaml (paste below)
ansible --version
```

### Expected Results

Should print the version

### Actual Results

```console
opt/homebrew/Cellar/ansible/5.7.0/libexec/lib/python3.10/site-packages/paramiko/transport.py:236: CryptographyDeprecationWarning: Blowfish has been deprecated
  "class": algorithms.Blowfish,
ansible [core 2.12.5]
  config file = None
  configured module search path = ['/Users/decades/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/homebrew/Cellar/ansible/5.7.0/libexec/lib/python3.10/site-packages/ansible
  ansible collection location = /Users/decades/.ansible/collections:/usr/share/ansible/collections
  executable location = /opt/homebrew/bin/ansible
  python version = 3.10.4 (main, Apr 26 2022, 19:36:29) [Clang 13.1.6 (clang-1316.0.21.2)]
  jinja version = 3.1.2
  libyaml = True
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by bcoca** [comment]
we should filter out on import (since that happens even when not used) but might want to issue it when it is actually used (connect() is called)

**Comment by TheBros35** [comment]
Can confirm the issue on the stated hardware platforms from  @neilyoung.

Looks like the next version of paramiko will have this issue fixed - patch appears to be related to changing a line of config:

https://github.com/paramiko/paramiko/issues/2038#issuecomment-1117345478

This could be backported into the module as well.

**Comment by tigattack** [comment]
Also seeing this on macOS 12.3.1 on Intel.

**Comment by solick** [comment]
current version of sensible seams to fix the problem.

**Comment by neilyoung** [comment]
Confirmed MacOS 12.4, Apple Silicon:

```
> ansible --version
ansible [core 2.12.5]
  config file = None
  configured module search path = ['/Users/decades/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/homebrew/Cellar/ansible/5.8.0/libexec/lib/python3.10/site-packages/ansible
  ansible collection location = /Users/decades/.ansible/collections:/usr/share/ansible/collections
  executable location = /opt/homebrew/bin/ansible
  python version = 3.10.4 (main, Apr 26 2022, 19:36:29) [Clang 13.1.6 (clang-1316.0.21.2)]
  jinja version = 3.1.2
  libyaml = True
```

Thanks for fixing it

**Comment by mattclay** [comment]
Re-opening this issue, since the only solution currently is to upgrade paramiko, which [doesn't work with older hosts](https://github.com/ansible/ansible/issues/76737).

**Comment by mattclay** [comment]
This has been resolved by https://github.com/ansible/ansible/pull/78789

The deprecation message is suppressed, and ansible has been updated so it can work with newer versions of paramiko, which do not trigger the warning.

**Comment by LuckyDacky** [comment]
Same problem **Ubuntu 22.04, ansible [core 2.13.4]**
Installed from ppa:ansible/ansible

```
/usr/lib/python3/dist-packages/paramiko/transport.py:236: CryptographyDeprecationWarning: Blowfish has been deprecated
  "class": algorithms.Blowfish,
ansible [core 2.13.4]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/antg/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /home/antg/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.4 (main, Jun 29 2022, 12:14:53) [GCC 11.2.0]
  jinja version = 3.1.2
  libyaml = True
```

**Comment by mattclay** [comment]
@LuckyDacky The fix is only available in the devel and stable-2.14 branches. The 2.14.0b1 release is out, if you don't mind trying out a pre-release.
